esp/clients/wsdfuaccess/wsdfuaccess.cpp

```diff
@@ -58,20 +59,20 @@ bool getFileAccess(StringBuffer &metaInfo, const char *serviceUrl, const char *j
     lfn.getCluster(cluster);
     lfn.get(lfnName); // remove cluster if present
 
-    dfuReq->setName(lfnName);
-    dfuReq->setCluster(cluster);
-    dfuReq->setExpirySeconds(expirySecs);
-    dfuReq->setAccessRole(CFileAccessRole_Engine);
-    dfuReq->setAccessType(translateToCSecAccessAccessType(access));
-    dfuReq->setJobId(jobId);
+    requestBase.setName(lfnName);
+    requestBase.setCluster(cluster);
+    requestBase.setExpirySeconds(expirySecs);
+    requestBase.setAccessRole(CFileAccessRole_Engine);
+    requestBase.setAccessType(translateToCSecAccessAccessType(access));
+    requestBase.setJobId(jobId);
 
     Owned<IClientDFUFileAccessResponse> dfuResp = dfuClient->DFUFileAccess(dfuReq);
 
     const IMultiException *excep = &dfuResp->getExceptions(); // NB: warning despite getXX name, this does not Link
     if (excep->ordinality() > 0)
         throw LINK((IMultiException *)excep); // JCSMORE - const IException.. not caught in general..
 
-    metaInfo.append(dfuResp->getMetaInfoBlob());
+    metaInfo.append(dfuResp->getAccessInfo().getMetaInfoBlob());
```

Comment:
do we need to ensure dfuResp->getAccessInfo() != nullptr?

Reply:
The getAccessInfo() is from generated code and will not be nullptr.

Reply:
References should never be null